pyproject.toml

```
@@ -1,3 +1,66 @@
+[tool.poetry]
+name = "latigo"
+version = "0.1.0"
+description = "A continuous prediction service that uses Gordo to predict data for IOC."
+authors = ["Equinor ASA <fg_gpl@equinor.com>"]
+license = "AGPL-3.0"
+packages = [
+    { include = "app" }
+]
+keywords=["gordo ioc continuous prediction"]
+
+# From https://pypi.org/pypi?%3Aaction=list_classifiers
+classifiers=[
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Other Audience",
+    "Topic :: Utilities",
+    "License :: OSI Approved :: GNU Affero General Public License v3",
+    "Natural Language :: English",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.7",
+    "Topic :: Other/Nonlisted Topic"
+]
+
+[tool.poetry.urls]
+"homepage"="https://github.com/equinor/latigo"
+
+[tool.poetry.dependencies]
+invoke = "^1.4.1"
+colorlog="^4.0"
+gordo="1.0.*"
+gordo-dataset="2.*"
+inject="4.2.0"
+pandas="^1.0"
+psycopg2-binary="^2.8"
+pylogctx="^1.12"
+requests="^2.20"
+requests_ms_auth="^0.*"
+confluent-kafka="^1.0.1"
+dataclasses-json="^0.3.5"
+setuptools="^41.0.0"
+opencensus-ext-azure="^1.0.2"
+redis="^3.5.3"
+ujson="^2.0.3"
+python = "^3.7"
```

does this mean that python3.8 is allowed?
Is yes, then only 3.7 should be supported.
I think that smth like this should do the trick but Im' not sure (taken from [here](https://python-poetry.org/docs/dependency-specification/#tilde-requirements))
```suggestion
python = "~3.7"
```

---

fixed